src/System.Globalization/tests/CultureInfo/CultureInfoCurrentCulture.cs

```diff
@@ -57,6 +57,7 @@ public void CurrentUICulture()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Thread cultures is not honored in UWP.")]
```

Comment:
It's totally fine to just make it no-op. We have done that in cases eg., where it configures some performance toggle so the caller will most likely not know we ignored the result. We just have to make a conscious decision. In this case I would think the caller would get a surprise if we didn't honor it so PNSE may be better.  @stephentoub  may have an opinion.

BTW that woudl be another issue not this PR.